HotFix/Core/Session.cs

```diff
@@ -51,6 +70,9 @@ public void Logon()
             Active = true;
         }
 
+        /// <summary>
+        /// Executes the session's logon process.
```

Comment:
This should be "logout". Not sure if it's worth saying that it can be safely called many times?